rancher-single-upgrade/rancher-single-upgrade.sh

```diff
@@ -0,0 +1,853 @@
+#!/bin/bash
+if hash tput 2>/dev/null; then
+    red=$(tput setaf 1)
+    green=$(tput setaf 2)
+    reset=$(tput sgr0)
+fi
+function grecho() {
+    echo "${green}$1${reset}"
+}
+function recho() {
+    echo "${red}$1${reset}"
+}
+function createbackupimagesfromvolumes() {
+    if [[ "${DOCKER_VOLUME_BASE}" == "" ]]; then
+        grecho "No directory was specified for docker volumes, using default of ${red}\"/var/lib/docker/volumes/\""
+        DOCKER_VOLUME_BASE="/var/lib/docker/volumes/"
+    fi
+    if [[ ! -d "${DOCKER_VOLUME_BASE}" ]]; then
+        grecho "The directory you specified does not exist!"
+        exit 1
+    fi
+    if [[ $EUID -ne 0 ]]; then
+        echo "Script must be run as the root user when using option -i."
+        exit 1
+    fi
+    #ARCHIVEDIR=/tmp/
```

Comment:
Can you remove this commented line please?

Reply:
fixed